batch/sc-batch-fargate.yaml

```diff
@@ -60,11 +60,10 @@ Parameters:
   Command:
     Type: String
     Description: >
-      A pipe separated list which is the command to pass to the container (i.e. echo|hello|world)
+      (Optional) A comma separated list which is the command to pass to the container (i.e. echo,hello,world).
```

Comment:
I believe that Docker creates the run command as a space separated string of the (pipe or comma separated) tokens you give it, e.g. "echo,hello,world" -> "docker run <image> echo hello world".  So if you passed "echo hello world,foo,bar" Docker would run "docker run <image> echo hello world foo bar".  So why now just let an entire command be the first and only element of the Command array?  Let the user enter "echo hello world" and simply pass that through to Fargate/Docker.  It would make the Command param' much more readable in the UI.

Reply:
I’m just not sure it’s the most desirable option.  Docker natively prefers a comma separated list for command so why should we go with space separated?  it’s documented right here: https://docs.docker.com/engine/reference/builder/#cmd
```
The CMD instruction has three forms:

CMD ["executable","param1","param2"] (exec form, this is the preferred form)
CMD ["param1","param2"] (as default parameters to ENTRYPOINT)
CMD command param1 param2 (shell form)
```
i’m not opposed to spaced separated i’m just wondering why docker prefers comma?  I'm guessing that there's a good reason for it therefore i prefer to pass in with the format that docker prefers.

Reply:
I vote shell form for humans

Reply:
I think theres a different when executing the command in the Dockerfile vs when doing `docker run`.  For instance, you wouldn't do:

```
docker run sagebionetworks/synapsepythonclient synapse,get,-r,syn1234
```

You would do
```
docker run sagebionetworks/synapsepythonclient synapse get -r syn1234
```

I vote shell form

Reply:
changed to shell form